### DISCUSSION TITLE
Release system

### DISCUSSION THREAD

**Discussion by noahfraiture** [zellij-org/zellij#3578] [Q&A]
Hello, I was wondering if it was possible to redraw the UI with a signal and saw that a PR have been merged to reload the config on runtime which is exactly what I want. But I also saw that the last release have been made in 4 months ago. What is the system for the release ? Is it twice a year or when there's a major change ? 
I don't NEED this so I don't think I will build locally, I prefer to stay on my package manager for now

**Comment by cristiand391** [comment]
> What is the system for the release ? Is it twice a year or when there's a major change ?

AFAIK there's to release schedule, so whenever Aram pushes the button 😄

**Comment by noahfraiture** [comment]
@imsnif Is there a release planned ?

**Reply by giyany** [reply]
There is always a release planned and there's never a due date. If you want updates on the next release, you can follow zellij on twitter or mastodon where it's immediately announced. It's also announced on the Discord and Matrix. Please refrain from tagging the lead maintainer for non-urgent matters, as they receive dozens of notifications daily, cannot and will not respond in this way.

**Reply by Lockszmith-GH** [reply]
@noahfraiture I personally shy away from the noise of Discord servers, and I track PRs and releases - I feel it is the right amount of noise-to-signal I can bare.

**Reply by noahfraiture** [reply]
I understand, thank you for your response and sorry for the ping. I guess I'll wait. I was simply surprised to see no news about a release in 5 months
